### PR TITLE
GRO-1: Render react artist follow buttons in mobile contexts

### PR DESCRIPTION
**Pull request by brainbicycle**
## Description

Fixes a bug preventing artist follow buttons from working in mobile context for editorial articles

- Adds a MobileFollowButton wrapped in a QueryRenderer 
- Parses html into react components for all cases and instead just doesn't add tooltips in mobile contexts

https://artsyproduct.atlassian.net/browse/FX-2282

### TODO:

- [ ] Link into positron to make sure this is working 
- [ ] Are there any cases we still need to just set raw html?  
- [x] Fix tests

### CODE REVIEW

File: src/Components/FollowButton/MobileFollowArtistButton.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  MobileFollowArtistButtonQueryResponse,
+  MobileFollowArtistButtonQueryVariables,
+} from "__generated__/MobileFollowArtistButtonQuery.graphql"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "./FollowArtistButton"
+import { ContextModule } from "@artsy/cohesion"
+
+export const MobileFollowArtistButton: React.FC<MobileFollowArtistButtonQueryResponse> = props => {
+  return (
+    <FollowArtistButton
+      contextModule={ContextModule.intextTooltip}
```
Comment (eessex) [inline src/Components/FollowButton/MobileFollowArtistButton.tsx:16]: Realizing this is not actually a tooltip -- @abhitip is there a context we should add for follow buttons within article text? 
Screen Shot 2020-10-01 at 6 07 26 PM

File: src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx [hunk 2]
```diff
@@ -100,6 +101,7 @@ export class Eoy2018Artists extends React.Component<EditorialFeaturesProps> {
           <Text
             html={section.body}
             layout={layout}
+            isMobile={isMobile}
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx:104]: We will probably want to use this in the `isTablet` use case as well.

Comment (brainbicycle) [reply src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx:104]: Good call, in that case maybe I can just reuse showToolTips and get rid of this prop

**Comment by jonallured** [comment]
Heyo! I am hijacking this PR - more chatter here:

https://artsy.slack.com/archives/C01ADJNCS5D/p1603399733040100

So that I can wrap this up for this one:

https://artsyproduct.atlassian.net/browse/GRO-1

First step was a rebase on latest master to see how things are looking.

File: src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx (hunk 2 above)
Comment (jonallured) [reply src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx]: This seems addressed but lmk if I've missed something @eessex!

File: src/Components/FollowButton/MobileFollowArtistButton.tsx (hunk 1 above)
Comment (abhitip) [reply src/Components/FollowButton/MobileFollowArtistButton.tsx:16]: Just added a new context module `articleArtist` and tagged @jonallured to merge! https://github.com/artsy/cohesion/pull/114

Comment (jonallured) [reply src/Components/FollowButton/MobileFollowArtistButton.tsx:16]: Merged! So my next step would be to wait for this to roll out and then use that new value in this PR, correct?

Comment (abhitip) [reply src/Components/FollowButton/MobileFollowArtistButton.tsx:16]: yup

File: src/Components/Publishing/Sections/Text.tsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ export class Text extends Component<Props, State> {
   static defaultProps = {
     color: "black",
     showTooltips: false,
+    isMobile: false,
```
Comment (brainbicycle) [inline src/Components/Publishing/Sections/Text.tsx:28]: I think this is unused now and we are just using showTooltips. Should be able to remove, I think I missed it 🙈

Comment (jonallured) [reply src/Components/Publishing/Sections/Text.tsx]: Done with d263255.

File: src/Components/FollowButton/MobileFollowArtistButton.tsx (hunk 1 above)
Comment (jonallured) [reply src/Components/FollowButton/MobileFollowArtistButton.tsx]: Done with fe24363.

**Comment by jonallured** [comment]
Ok I think this is ready for final review @eessex!

**Comment by jonallured** [comment]
Ok I merged this one pals!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v28.13.0`](https://github.com/artsy/reaction/releases/tag/v28.13.0) :rocket:

**Pull request merged**